Platform/TigerlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -279,6 +279,13 @@
                      0- Disable, 1- Enable (Default) Azalia controller
       length       : 0x01
       value        : 0x01
+  - PortUsb20Enable :
+      name         : Enable USB2 ports
+      type         : EditNum, HEX, (0x00,0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF)
+      help         : >
+                     Enable/disable per USB2 ports. One byte for each port, byte0 for port0, byte1 for port1, and so on.
+      length       : 0x10

```

Comment:
Can we use 1 bit per port for CFGDATA to save size ?

Reply:
Please see latest patch, if there is some other example of a way to achieve 1-bit per port in CFG data please advise. I tested the change and did confirm the correct values are being assigned in the latest patch.